examples_tests/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ add_subdirectory(23.Autoexposure EXCLUDE_FROM_ALL)
 #add_subdirectory(24.BufferSubDataAlignTest EXCLUDE_FROM_ALL) # TODO: @Anastazluk or @Hazardu
 #add_subdirectory(25.Blur EXCLUDE_FROM_ALL)
 add_subdirectory(26.MultidrawIndirectVSCPUCull EXCLUDE_FROM_ALL)
+add_subdirectory(27A.LoaderFixes EXCLUDE_FROM_ALL)
```

Comment:
just use number 24, we dont need buffer sub align test

Reply:
corrected in following commit